README.rst

```diff
@@ -36,15 +36,13 @@ of results.
 
 The workflow is based on `Nipype <https://nipype.readthedocs.io>`__ and encompases a large
 set of tools from well-known neuroimaging packages, including
-`FSL <https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/>`__,
 `ANTs <https://stnava.github.io/ANTs/>`__,
 `FreeSurfer <https://surfer.nmr.mgh.harvard.edu/>`__,
-`AFNI <https://afni.nimh.nih.gov/>`__,
-and `Nilearn <https://nilearn.github.io/>`__.
+and `DIPY <https://dipy.org/>`__.
```

Comment:
Should we just not mention any of these here (so long the citations are clearly stated in the boilerplate)?

Reply:
Yes. I think that would be fine. Implemented in 5431efc